database/dbops.js

```diff
@@ -13,6 +15,7 @@ const VID_TYPES = {
 }
 
 const N_LEVELS_PER_NEW_LIFE = 50;
+const MAX_LEVEL_TIME_SEC = 1 * 60 * 60; // 1 hour
```

Comment:
Seem reasonable to me; we might want to drop this to half an hour in the future if we realize that stuff gets out of sync quickly.

Reply:
Possibly. I wanted to be sure I wasn't going to give anyone with a slow connection trouble. But an hour might be overkill.